modus/overview.mdx

```diff
@@ -8,50 +8,72 @@ sidebarTitle: "Overview"
 
 ## What is Modus? {/* vale Google.Contractions = NO */}
 
-Modus is an open source, serverless framework for building functions and APIs,
-powered by WebAssembly.
+Modus is an open source agent framework (including both a runtime and SDK) built
```

Comment:
@johnymontana The fact that we mention WebAssembly so soon in this introduction could give the impression that you need to know WebAssembly to proceed. 

I would say something like: 

Modus is an open source agent framework (including both a runtime and SDK) built for Go and AssemblyScript, compiled to WebAssembly.